index.bs

```diff
@@ -686,16 +686,71 @@ dictionary MLOperandDescriptor {
 
 ### The MLOperand interface ### {#api-mloperand}
 
-An {{MLOperand}} represents an intermediary graph being constructed as a result of compositing parts of an operation into a fully composed operation.
+An {{MLOperand}} represents an intermediary graph edge being constructed as a result of compositing parts of an operation into a fully composed operation.
 
 For instance, an {{MLOperand}} may represent a constant feeding to an operation or the result from combining multiple constants together into an operation. See also [[#programming-model]].
 
 <script type=idl>
+enum MLOperandKind {"constant", "input", "output"};
```

Comment:
I'm not sure I understand the reason to add the new `MLOperandKind` here. Has there been an issue opened for this or a discussion in a WG meeting for the need of it?

Reply:
Thanks Chai for reviewing. An issue is indeed missing ATM, because the dependency of `kind` is future work.

The `kind` is an internal slot of `MLOperand` used in the build steps (currently missing from the spec, but I have a draft for it) to check if the current `operand` in a `graph` is a "constant" or an "input" kind. If it's an input, it's added to 
`|graph|.{{MLGraph/[[inputDescriptors]]}}`, otherwise if it's constant, it's preprocessed/optimized.

It is only an internal slot, hence we don't need IDL definitions for it indeed. We can just use "constant" and "input" as strings in the spec prose.
But we'd need to keep the internal slot - or otherwise please suggest how to discern in algorithmic steps the operands in a graph to be constant or input.
It's also used in the implementation.

We can discuss this in the next call.

Reply:
Thanks for clarifying. If it's meant for internal slot only, please remove it from the IDL.